components/Card.css

```
@@ -0,0 +1,134 @@
+.card {
+  padding: 1rem;
+  width: 300px;
+  border-radius: 0.5rem;
+  background-color: var(--text-lighter);
+}
+/* .card:hover {
+  transform: scale(1.1);
+  transition: 0.2s;
+} */
+
+.card-title {
+  font-weight: 800;
+  margin-bottom: 1rem;
+}
+.card-description {
+  color: var(--text-gray-400);
+}
+
+/* Card without dismiss */
+.card-ecom {
+  width: 240px;
+  position: relative;
+  overflow: hidden;
+  padding: 0;
+}
+.card-ecom:hover {
+  box-shadow: 1px 1px 40px 5px rgba(0, 0, 0, 0.05);
+}
+.card-img-cont {
+  background-color: #fff4f8;
+  width: 100%;
+  height: 280px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  overflow: hidden;
+}
+.card-img-cont img {
+  width: 80%;
+}
+.card-ecom .card-badge {
+  position: absolute;
+  top: 1rem;
+  left: 0;
+  padding: 0.1rem 0.5rem;
+  height: 24px;
+  background-color: var(--primary-dark-color);
+  color: var(--primary-light-color);
+  font-weight: 600;
+}
+.card-ecom .card-body {
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+.card-ecom .card-title {
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  margin-bottom: 0.5rem;
+}
+.card-ecom .card-desc {
+  font-size: 16px;
+  line-height: 16px;
+  font-weight: 400;
+  margin-bottom: 0.5rem;
+  color: var(--primary-light-color);
```

`.card-desc` text is a bit difficult to read. Try to use darker color or a lighter background.

---

```suggestion
  color: var(--primary-light-color);
```
Thankyou. Will do that